cli/commands/benchmark-web-vitals.mjs

```diff
@@ -180,55 +303,38 @@ export async function handler( opt ) {
 }
 
 /**
- * @param {Browser} browser
- * @param {Params}  params
+ * @param {string}                 url
+ * @param {Browser}                browser
+ * @param {MetricsDefinitionEntry} metricsDefinition
+ * @param {Params}                 params
```

Comment:
```suggestion
@param {string}                                 url
@param {Browser}                                browser
@param {Object<string, MetricsDefinitionEntry>} metricsDefinition
@param {Params}                                 params
```

Reply:
Why the different comment format? I followed the one already present above. Is that also incorrect?

Reply:
Different format because it's a different value, right? Here it's passing the object mapping keys to the individual `MetricsDefinitionEntry` objects. Without this change, TypeScript was alerting me of a type error.

![image](https://github.com/GoogleChromeLabs/wpp-research/assets/134745/1fd31672-fa0b-42af-a287-9e7fb406da68)

So the `@param` type here should match whatever is returned by `getMetricsDefinition()`, right?

Reply:
@westonruter I was referring to you removing the ` * ` from each comment line. The `@param {Object<string, MetricsDefinitionEntry>} metricsDefinition` makes sense to me, but why remove the ` * `?

Reply:
Oh sorry, that was a mistake.

Reply:
Updated in https://github.com/GoogleChromeLabs/wpp-research/pull/78/commits/6ebf8019c4fb7a1ce508c46cae2311cdc1d028bc